sdk/core/Azure.Core/README.md

```diff
@@ -82,9 +83,25 @@ SecretClient client = new SecretClient(new Uri("http://example.com"), new Defaul
 
 More on client configuration in [client configuration samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Configuration.md)
 
+### Customzing Request Using `RequestContext`
+
+Besides general configuration of _serice client_ through `ClientOptions`, you can customize each individual request sent by _service client_
+using `RequestContext`.
+
+_Service clients_ have methods that can be used to call Azure services. We refer to these client methods _service methods_. You can
+pass `RequestContext` to each _service method_ as a parameter.
```

Comment:
Suggest combining this sentence with the previous paragraph. 

```suggestion
Each _service client_ method involving a call to an Azure service accepts a `RequestContext` instance as a parameter.
```

Reply:
We should remove this, because not all service client methods expose `RequestContext`.

Reply:
I move the sentence regarding _service methods_ back to section "Accessing HTTP Response Details Using ```Response<T>```".